command/e2etest/automation_test.go

```diff
@@ -41,11 +40,11 @@ func TestPlanApplyInAutomation(t *testing.T) {
 
 	// Make sure we actually downloaded the plugins, rather than picking up
 	// copies that might be already installed globally on the system.
-	if !strings.Contains(stdout, "- Downloading plugin for provider \"template\"") {
+	if !strings.Contains(stdout, "- Downloading plugin for provider \"template") {
```

Comment:
I think #21094 has removed the need for this change, the similar change below for `null`, and similar stuff in other tests. Won't do any harm to still include it, but maybe cleaner to leave it as it was now that we've restored (approximately) the old formatting anyway.

Reply:
Sorry, didn't mean to resolve this!
These changes are still required; here's the actual output:
```- Downloading plugin for provider "null (terraform-providers/null)" (2.1.1)...```

Reply:
I can either leave this as-is, or update the test with the full line, if you have strong feelings in either direction.